### DISCUSSION TITLE
Editing daemon.json *nvm, using postfix

### DISCUSSION THREAD

**Discussion by 99isfailure** [Security-Onion-Solutions/securityonion#5844] [Show and tell]
I'm attempting to install mailcow on my SO VM to send alerts if/when triggered. I'm currently stuck at the step to modify the /etc/docker/daemon.json file and add the "selinux-enabled : true" line. I should be getting a return when running "docker info | grep selinux" after updating the file but I am not. Also, when I restart the machine the daemon.json file goes back to the default configuration. Any help would be appreciated.
![somailhelp](https://user-images.githubusercontent.com/90627776/136845137-6ee54e11-d7bc-43d5-9951-4fe4cdccb036.PNG)

**Comment by weslambert** [comment]
Can you let me know why you need to enable `selinux-enabled: true`?  What kind of alerts/mail are you looking to receive?

**Reply by 99isfailure** [reply]
It's from a tutorial I was following https://mailcow.github.io/mailcow-dockerized-docs/i_u_m_install/
My end-goal is to receive alerts from Security Onion when a certain risk level is triggered (ex. emailed when there's a critical alert).
